action.yml

```diff
@@ -5,6 +5,9 @@ branding:
   color: blue
   icon: anchor
 inputs:
+  verify_blob:
+    description: "determines whether the download blob should be verified (default: true)"
+    default: 'true'
```

Comment:
add the `required: false`


can you also make sure that will work like `verify_blob: false` and `verify_blob: 'false'` ?

Reply:
Tested both cases with [act](https://github.com/nektos/act) and received the [messages](https://github.com/helm/chart-testing-action/pull/131/commits/6acc8cc67bee951c83257172c69f7c6d7d503ee1) from the log as expected -> works with both (with and without quotes).